check_logs.py

```diff
@@ -26,31 +26,28 @@ def _fetch(title, url, dest):
             urllib.request.urlretrieve(url, dest)
             break
         except ConnectionResetError as err:
-            print_yellow('%s download error ("%s"), retrying...' %
-                         (title, str(err)))
+            print_yellow(f'{title} download error ("{str(err)}"), retrying...')
```

Comment:
while you're here, mind changing this to double quotes? Let's be consistent throughout the codebase.

Reply:
5a4758e7e3ec8d7e70afc20ab0c30b1ea95dbd70